packages/coinstac-api-server/src/data/resolvers.js

```diff
@@ -1877,7 +1878,7 @@ const resolvers = {
     runWithHeadlessClientStarted: {
       subscribe: withFilter(
         () => pubsub.asyncIterator('runWithHeadlessClientStarted'),
-        (payload, variables) => (variables.clientId && variables.clientId in payload.runWithHeadlessClientStarted.clients)
+        (payload, variables) => (variables.clientId && variables.clientId in payload.runWithHeadlessClientStarted.pipelineSnapshot.headlessMembers)
```

Comment:
I think the the other way is the right one (using `.clients`)

Reply:
Yeah I don't know how that change even got there. Not something I did intentionally.